posthog/models/annotation.py

```diff
@@ -12,7 +13,7 @@ class CreationType(models.TextChoices):
         GITHUB = "GIT", "GitHub"
 
     content: models.CharField = models.CharField(max_length=400, null=True, blank=True)
-    created_at: models.DateTimeField = models.DateTimeField(null=True, blank=True)
+    created_at: models.DateTimeField = models.DateTimeField(default=timezone.now, null=True)
```

Comment:
Could we use `auto_now_add` instead? This sets the timestamp directly in the DB instead of getting a value from django, ensuring that it's always there.

`created_at: models.DateTimeField = models.DateTimeField(auto_now_add=True, blank=True)`

Reply:
I had to change it to the `default=` approach due to test `test_query_annotations_by_datetime`, which needs `created_at` to be settable.

Reply:
aha, fair enough then I guess